packages/terra-form-input/src/InputField.jsx

```diff
@@ -176,6 +190,7 @@ const InputField = (props) => {
     >
       <Input
         {...inputAttrs}
+        className={classes}
```

Comment:
you have to continue pulling forward classNames from inputAttrs
```suggestion
        className={`${classes} ${inputAttrs.className}`}
```

Reply:
No longer an issue now since `isIncomplete` works more similar to the `isInvalid` prop and the Input control itself adds the class in. https://github.com/cerner/terra-core/pull/2670/commits/93158da0ccc2be1d72588890924879e913215a96